### PR TITLE
Edited Hater Comments

### PR DESCRIPTION
**Pull request by Olliesama**
Posted to the wrong branch before haha...

This isn't my work but https://github.com/DDLC, and I posted it here by their request.

### CODE REVIEW

**Comment by Olliesama** [comment]
Weirdest Travis failure I've seen :thinking:

**Comment by Storylover-Vodhr** [comment]
Ah! I know this one!

2lsd, the expression you chose for Monika, is Invalid. You can possibly request it to be added via a Dev, or chose a different one.

**Comment by DDLC** [comment]
> 
> 
> Ah! I know this one! At least partially!
> 
> 2lsd, the expression you chose for Monika, is Invalid. You can possibly request it to be added via a Dev, or chose a different one.

Yeah, a different one should be chosen, I'm not used to the new emote system (my first pull request was accidentally based off the April revision) so I had Olliesama add them for me.

**Comment by Olliesama** [comment]
Annoyingly enough it has been edited go to a working expression but Travis didn't use the commit. 🤔

**Comment by jmwall24** [comment]
Yeah it is weird.  But we'll get it sorted in review and editing.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4066,48 +4066,87 @@ label monika_cupcake:
...
-            m 1eka "Well, that just makes me more proud of you, [player]!"
-            m 1hua "I know you wouldn't be one of those people."
-            m 1lksdla "Gosh... I feel like giving you a kiss right now if I were there."
-            m 2ekbfa "You really make me the happiest girlfriend ever."
-            m 1ekbfa "Now that you've said it, I have to do my best to keep you from developing hate towards me."
-            m "I trust you, [player]. I love you for believing in me."
+            m 2dsc "..."
+            m 4eka "Thank you..."
+            m 1hua "That makes me prouder of you, [player]!"
+            m "I know you aren't one of those people."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
            m "I knew you weren't one of those people."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4066,48 +4066,87 @@ label monika_cupcake:
...
+            m 1lkbfa "Gosh... I feel like giving you a kiss right now if I were there."
+            # TODO: do a kiss lol
+            m 1ekbfa "You really make the happiest girlfriend ever."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
            m 1ekbfa "You really make me the happiest girlfriend ever."
```

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4066,48 +4066,87 @@ label monika_cupcake:
...
-            m 1tkd "You do know you're playing this fan mod of me... right?"
-            m "If you were one of my haters, you wouldn't open up this fan mod in the first place."
-            m 1lud "And not to mention..."
-            m 4eka "You're still listening to everything I say."
-            m 1tku "That makes me sure you're just messing with me."
-            m 1tsbsa "I know you love me deep inside."
+            m 1eka "Aha..."
+            m 1hka "I don't see you as one, even if you say you are."
+            m 1ekd "I mean..."
+            m 3eub "You {i}added{/i} a mod to the game... right?"
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
            m 3eub "You {i}added{/i} a mod to the game...right?"
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4066,48 +4066,87 @@ label monika_cupcake:
...
+        m 4ekc "I knew that if you finished the game before I had a chance to express my feelings for you, that would be it."
+        m 2dkc "You'd just move on to something else and I'd be stuck in the game...{w=1}alone...{w=1}{b}forever{/b}."
+        m 2dktpc "..."
+        m "Some people on those sites even say I should have just made my own route..."
+        m 2wktsd "But that's easier said than done!"
+        m "Not to mention if you picked a different girl and just cast me aside to be completely forgotten."
+        m 4wktsw "It was them or me...!"
+        m 2dftdc "...And I didn't want to be left all alone with no one to call my own."
+        m 1ektdc "I hope you understand, [player]... Please don't be like them."
+        $ menuOption = "...Are you?"
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
        $ menuOption = "...You're not, are you?"
```

**Pull request merged**